runtime-modules/proposals/codex/src/types.rs

```diff
@@ -96,43 +136,43 @@ pub struct GeneralProposalParams<MemberId, AccountId, BlockNumber> {
 #[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
 #[derive(Encode, Decode, Clone, PartialEq, Debug)]
 pub struct TerminateRoleParameters<WorkerId, Balance> {
-    /// Leader worker id to fire.
+    /// Worker identifier.
     pub worker_id: WorkerId,
 
-    /// Terminate role slash penalty.
-    pub penalty: Option<Penalty<Balance>>,
+    /// Optional amount to be slashed.
```

Comment:
I think that the gitbook should be update rather than changing this parameter.

Reply:
I think this parameter is missing in the gitbook because the generic description for the proposal already doubles as a rationale. I remember bedeho saying he dropped the rationale from slashing lead for this reason.

Reply:
Ok. Make a comment please when you pass the empty text on two occasions: slashing and termination.